spec/models/featured_collection_list_spec.rb

```diff
@@ -35,6 +36,20 @@
         expect(presenter.id).to eq collection2.id
       end
     end
+
+    context 'when the featured collections have not been manually ordered' do
+      it 'is sorted by title' do
+        allow(subject).to receive(:manually_ordered?).and_return(false)
```

Comment:
Another lint errors on stubbing subjects. Looking closer at this test, I don't think you will need to stub it at all if you implement either suggestion from above. 

Another semi related idea, you could test whether or not each context receives :sort_by_title

Reply:
i incorporated the suggestion and removed the `let!` but still needed to somehow control what `#manually_sorted` returned, please take a look